test/Combinatorics/EnumerativeCombinatorics/combinations.jl

```diff
@@ -51,6 +51,17 @@
 
   @test collect(combinations(0, 0)) == [Int[]]
 
+  @testset "inplace iteration" begin
+    Ci = combinations(5,3, inplace=true)
+    Cf = combinations(5,3)
+    si = sf = nothing
+    for i in 1:number_of_combinations(5,3)
+      ci, si = iterate(Ci, si)
+      cf, sf = iterate(Cf, sf)
+      @test ci == cf
+    end
```

Comment:
```suggestion
		@test all(splat(==), zip(Ci, Cf))
```
you could also do this, as `zip` evaluates its arguments lazily (and of course the same for the other two test files)